src/commands/doctor.ts

```diff
@@ -110,6 +125,7 @@ module.exports = {
       ...pnpmPackages,
       bunInfo,
       expoInfo,
+      ...(depExpoCLIInfo ? [depExpoCLIInfo] : []),
```

Comment:
```suggestion
      depExpoCliInfo,
```

I believe table knows what to do with a null value here, so we can just simplify this up!

Reply:
I don't believe you are correct here. And I was right I tested it with your change and the result was

```bash
ignite-cli doctor
System
  platform           darwin
  arch               arm64
  cpu                8 cores      Apple M2
  directory          ignite       /Users/dax/Code/contribs/ignite

JavaScript (and globally-installed packages)
/Users/dax/Code/contribs/ignite/node_modules/gluegun/node_modules/cli-table3/src/layout-manager.js:143
    return rows.map(function (row) {
                ^
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at /Users/dax/Code/contribs/ignite/node_modules/gluegun/node_modules/cli-table3/src/layout-manager.js:145:26
    at Table.map (<anonymous>)
    at generateCells (/Users/dax/Code/contribs/ignite/node_modules/gluegun/node_modules/cli-table3/src/layout-manager.js:143:17)
    at Object.makeTableLayout (/Users/dax/Code/contribs/ignite/node_modules/gluegun/node_modules/cli-table3/src/layout-manager.js:161:20)
    at Table.toString (/Users/dax/Code/contribs/ignite/node_modules/gluegun/node_modules/cli-table3/src/table.js:23:29)
    at table (/Users/dax/Code/contribs/ignite/node_modules/gluegun/src/toolbox/print-tools.ts:136:17)
    at Command.<anonymous> (/Users/dax/Code/contribs/ignite/src/commands/doctor.ts:118:5)
    at step (/Users/dax/Code/contribs/ignite/src/commands/doctor.ts:33:23)
    at Object.throw (/Users/dax/Code/contribs/ignite/src/commands/doctor.ts:14:53)
    ```
    I can leave it as is, create another PR to fit the table handling a null case, since its from Gluegun I can check the docs to see if does have info on gracefully handling null otherwise we can revert please advise.